python/src/robyn/allocator/allocation_optimizer.py

```diff
@@ -1,18 +1,24 @@
+#pyre-strict
+"""Optimization module for budget allocation."""
+
 from typing import Callable, List, Tuple, Dict
+import logging
 import numpy as np
 from scipy.optimize import minimize
 from robyn.allocator.entities.enums import ConstrMode
 
-
 class AllocationOptimizer:
     """Handles numerical optimization for budget allocation."""
 
     def __init__(self):
         """Initialize optimizer."""
         self.supported_methods = {
-            "SLSQP_AUGLAG": "SLSQP",  # Map Robyn method names to scipy method names
-            "MMA_AUGLAG": "SLSQP",  # For now, map both to SLSQP as it's most similar
+            "SLSQP_AUGLAG": "SLSQP", #TODO: both constant values are same. is this correct?
```

Comment:
@alxlyj Not related to this logging PR but are these values correct? Both constants have same value.